eq-author-api/schema/typeDefs.js

```diff
@@ -24,6 +24,11 @@ type QuestionnaireInfo {
   totalSectionCount: Int!
 }
 
+type PublishDetails {
+  surveyId: String
+  formType: JSON
+}
+
```

Comment:
This shouldn't be here; looks like something I should have removed from one of my PRs accidentally got forgotten about by myself. If you could remove it, that would be grand.

Reply:
Somehow one of your commits has ended up in this - maybe during a rebase?
I have reverted your commit and that looks to have addressed all the comments below...